CRM/Price/BAO/LineItem.php

```diff
@@ -477,6 +477,7 @@ public static function processPriceSet($entityId, $lineItem, $contributionDetail
         }
       }
     }
+    // create deferred revenue transaction
```

Comment:
This comment doesn't really tell use anything does it? We are calling createDeferredTrxn